recipe/meta.yaml

```diff
@@ -10,32 +10,42 @@ source:
   url: https://github.com/pygobject/pycairo/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
   sha256: {{ sha256 }}
   patches:
-    - windows.patch  # [win]
+    - typing-test-workaround.patch
 
 build:
   number: 0
   detect_binary_files_with_prefix: true
-  skip: true  # [py < 35]
+  skip: true  # [py < 37]
+  ignore_run_exports:
+    - glib
+    - python
+    - pixman
 
 requirements:
   build:
-    - pkg-config  # [not win]
+    - python
+    - pkg-config
+    - meson >=0.53.0
     - {{ compiler('c') }}
-    - m2w64-toolchain  # [win]
+    - pthread-stubs                      # [linux]
+    - {{ cdt('libxau-devel') }}          # [linux]
+    - {{ cdt('libxau') }}                # [linux]
+    - {{ cdt('libxcb') }}                # [linux]
+    - {{ cdt('xorg-x11-util-macros') }}  # [linux]
     - {{ cdt('xorg-x11-proto-devel') }}  # [linux]
     - {{ cdt('libx11-devel') }}          # [linux]
-    - {{ cdt('libxcb') }}                # [linux]
-    - {{ cdt('libxau') }}                # [linux]
-    - {{ cdt('libxau-devel') }}          # [linux and aarch64]
-    - {{ cdt('libxext-devel') }}         # [linux and aarch64]
-    - m2w64-pkg-config                   # [win]
-    - m2-patch    # [win]
+    - {{ cdt('libx11-common') }}         # [linux]
+    - {{ cdt('libx11') }}                # [linux]
+    - {{ cdt('libxext-devel') }}         # [linux]
+    - {{ cdt('libxext') }}               # [linux]
+    - {{ cdt('libxrender-devel') }}      # [linux]
+    - patch     # [not win]
+    - m2-patch  # [win]
   host:
-    - m2w64-pkg-config  # [win]
-    - cairo
-    - cython
-    - pip
-    - pthread-stubs
+    - cairo >=1.15.10
+    - flake8  # for testing during the build
+    - glib
+    - pytest  # for testing during the build
     - python
```

Comment:
It requires `setuptools` and `wheel` in `host`, see https://github.com/pygobject/pycairo/blob/7f7869f4139dbcc84a6f688a321516cb25cea1b0/pyproject.toml#L21